Tests/IntegrationTests/Card Tests/CardComponentEventTests.swift

```diff
@@ -33,8 +33,20 @@ final class CardComponentEventTests: XCTestCase {
         // Then
         XCTAssertEqual(analyticsProviderMock.initialEventCallsCount, 1)
         XCTAssertEqual(analyticsProviderMock.infos.count, 1)
-        let infoType = analyticsProviderMock.infos.first?.type
-        XCTAssertEqual(infoType, .rendered)
+        
+        let info = analyticsProviderMock.infos.first
+        XCTAssertEqual(info?.type, .rendered)
+        
+        let configDataDict = info?.configData?.stringOnlyDictionary
```

Comment:
maybe it would be interesting to also check for the count so we know if "encoding" changed